Documentation/docs/learn/faq.md

```diff
@@ -0,0 +1,561 @@
+| [STL](https://en.wikipedia.org/wiki/STL_%28file_format%29) | | |
+| [SWC Neuron Morphology](https://swc-specification.readthedocs.io/en/latest/) | | |
+| [OBJ](https://en.wikipedia.org/wiki/Wavefront_.obj_file) | [`ìtk::OBJMeshIO`](https://itk.org/Doxygen/html/classitk_1_1OBJMeshIO.html) | |
+| [VTK legacy file format for vtkPolyData](https://www.vtk.org/wp-content/uploads/2015/04/file-formats.pdf) | [`itk::VTKPolyDataMeshIO`](https://itk.org/Doxygen/html/classitk_1_1VTKPolyDataMeshIO.html) | |
+
+More information on supported mesh and point set file formats is available on the
+[ITK-wasm Mesh and Point Set File Formats](https://wasm.itk.org/en/latest/introduction/file_formats/meshes.html)
+site.
+
+### What transform file formats can ITK import and export?
+
+ITK supports reading and writing the following transform file formats:
+
+| File Format | Read/Write | Import |
+| ----------- | ---------- | ------ |
+| InsightLegacy | [`itk::TxtTransformIOTemplate`](https://itk.org/Doxygen/html/classitk_1_1TxtTransformIOTemplate.html) | |
+| ITK HDF5 | [`itk::HDF5TransformIOTemplate`](https://itk.org/Doxygen/html/classitk_1_1HDF5TransformIOTemplate.html) | |
+| MATLAB | [`itk::MatlabTransformIOTemplate`](https://itk.org/Doxygen/html/classitk_1_1MatlabTransformIOTemplate.html) | |
+| MINC (Medical Image NetCDF) | `itk::MINCTransformIOTemplate` | |
+
+#### DICOM data
+
+##### How do I read a volume from a DICOM series?
+
+The following are alternative options for reading DICOM series. The
+first one in the list is the recommended option. The others are provided
+in case of desperation.
+
+1. Use the [itk-wasm dicom](https://itk-wasm-dicom-docs.on.fleek.co/#/),
+   package, which is available from [here](https://itk-wasm-dicom-python-docs.on.fleek.co/).
+   It has the broadest support and is the easiest to use.
+1. Use the [`itk::ImageSeriesReader`](https://itk.org/Doxygen/html/classitk_1_1ImageSeriesReader.html) in combination with the
+   `DicomSeriesFileNames`. For a full example on how to do this,
+   please look at the [`DicomSeriesReadImageWrite2.cxx`](https://github.com/InsightSoftwareConsortium/ITK/blob/master/Examples/IO/DicomSeriesReadImageWrite2.cxx)
+   code.
+1. Write a MetaImage header. This is a small text file holding
+   information about the image: spacing, dimensions, pixelt type, etc.
+   This header can hold the list of DICOM files you want to read. The
+   only restriction is that the files must be uncompressed.
+   The easy way to create a MetaImage header is to use the application:
+   `MetaImageImporter` that is available in
+   [ITKApps/MetaImageImporter](https://github.com/InsightSoftwareConsortium/ITKApps/tree/master/MetaImageImporter).
+   You can also take an existing MetaImage reader and modify it in
+   order to fit your needs. Please find attached one file that you
+   can edit it with your favorite text editor.
```

Comment:
As per https://github.com/InsightSoftwareConsortium/ITK/pull/4313#discussion_r1390272297, mentions to `ITKApps` should be removed. @thewtex ?

Reply:
Yes, ITKApps is no longer maintained. I do no think we should direct people to it.

Reply:
Done in PR #4364.